03week/ticTacToe.js

```diff
@@ -23,25 +32,109 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
-function horizontalWin() {
-  // Your code here
+//rules for every horizontal win
+const horizontalWin=()=> {
+  console.log('playerTurn=',playerTurn)
+  if(board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn) {
+    return true;
+  } else if (board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn) {
+    return true;
+  } else if (board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn) {
+    return true;
+  } else {
+    return false;
+  }
 }
 
-function verticalWin() {
-  // Your code here
+//rules for every vertical win
+const verticalWin=()=>{
+  if(board[0][0] === playerTurn && board[1][0] === playerTurn && board[2][0] === playerTurn) {
+    return true;
+  } else if (board[0][1] === playerTurn && board[1][1] === playerTurn && board[2][1] === playerTurn) {
+    return true;
+  } else if (board[0][2] === playerTurn && board[1][2] === playerTurn && board[2][2] === playerTurn) {
+    return true;
+  } else {
+    return false;
+  }
 }
 
+//rules for every diaganol win
 function diagonalWin() {
-  // Your code here
+  if(board[0][0] === playerTurn && board[1][1] === playerTurn && board[2][2] === playerTurn) {
```

Comment:
you're repeating yourself a lot here, just return the evaluation. How else could you simplify?